code/script.js

```diff
@@ -1,213 +1,252 @@
-// All the DOM selectors stored as short variables
-const board = document.getElementById('board')
-const questions = document.getElementById('questions')
-const restartButton = document.getElementById('restart')
+// Ylva - All the DOM selectors stored as short variables
+const board = document.getElementById("board");
+const questions = document.getElementById("questions");
+const restartButton = document.getElementById("restart");
+const findOutButton = document.getElementById("filter");
+const winOrLoseText = document.getElementById("winOrLoseText");
+const winOrLoseSection = document.getElementById("winOrLose");
+const playAgainButton = document.getElementById("playAgain");
+const secretImageAtCheck = document.getElementById("secretImage");
+const numberOfQuestionsAllowed = document.getElementById("numberOfQuestions");
+const gameTimer = document.getElementById("timer");
 
-// Array with all the characters, as objects
+// Ylva - Array with all the characters, as objects
 const CHARACTERS = [
```

Comment:
I think the main reason that your game doesn't work is the change you made in the name of the categories (ie "hair" to "Hair"). All of the character objects still have the key as "hair" and "eyes", but the game is looking for the value of "Hair" or "Eyes". Since that category does not exist in the character objects (it is case sensitive) checkQuestion will always return false and filterCharacters will not filter out anyone.

Reply:
@edv-rd Yeah Michelle told me that in a comment on Slack! But here code works and she doesn't have the "Hair" and "Eyes" with the first letter big under the objects - and her code works... I think I know what you mean though.
That the optgroup label in HTML has to match the category in JS - right?
HTML example:
<optgroup label="Hair">
JS example (only a piece but I think you can see what I mean :) 
(category === "**Hair**" || category === "**Eyes**")